projects/sport-data-valley/seeds/questionnaires/psychological_competencies.rb

```diff
@@ -0,0 +1,416 @@
+      en: ""
+    }
+  }, {
+    section_start: 'Psychologische competentievragenlijst',
+    id: :v1,
+    type: :radio,
+    title: 'Ik verklaar op een voor mij duidelijke wijze te zijn ingelicht over aard, methode, doel, risico’s en de belasting van het onderzoek. Ik ga ermee akkoord dat de verzamelde gegevens en resultaten gebruikt kunnen worden voor wetenschappelijk onderzoek.',
```

Comment:
so you disabled the informed consent at the protocol level, and made it part of the questionnaire itself. Is it because they needed something custom?

Reply:
Sot this is a straight import of a copy of a questionnaire that they had in a different system, where they solved this in the way that it is now. I think ideally we would split up the informed consent and questionnaire into separate things, but the way it's phrased here, with people being unable to continue if they didn't check all three boxes I think is fine.